womtool/src/test/resources/validate/wdl_draft2/valid/relative_local_import/relative_local_import.wdl

```diff
@@ -0,0 +1,11 @@
+import "./womtool/src/test/resources/validate/wdl_draft2/valid/task_only/task_only.wdl" as imports
+
+workflow http_import {
+
+  File in_file
+
+  call imports.task_only {
+    input:
+      in_file = in_file
+  }
+}
```

Comment:
Request to please add a newline to these two files!

Reply:
Done!